ports/d3d12-memory-allocator/vcpkg.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "d3d12-memory-allocator",
+  "version": "1.0.0",
+  "port-version": 1,
```

```suggestion
  "port-version": 0,
```
The value should be 0 for new ports.